app/views/communities/show.html.erb

```diff
@@ -2,54 +2,85 @@
   <%= image_tag(@community.cover_image, height: "200px", width: "1200px") %>
 <% else %>
   <%= link_to "#" do %>
-    <img height= "200px", width= "1200px" src="https://cdn.cbeditz.com/cbeditz/preview/solid-dark-blue-gradient-background-wallpapers-70-116145018144hkixikfuh.jpg">
+    <img height= "170px", width= "100%" src="https://cdn.cbeditz.com/cbeditz/preview/solid-dark-blue-gradient-background-wallpapers-70-116145018144hkixikfuh.jpg">
```

Comment:
if this is the cover image, instead of creating this using img tag can you add it through css?

Reply:
also don't use external url for assets, instead download the image and load it locally